src/seedu/addressbook/data/AddressBook.java

```diff
@@ -38,6 +44,16 @@ public void addPerson(Person toAdd) throws DuplicatePersonException {
         allPersons.add(toAdd);
     }
 
+    /**
+     * Adds a person to the favorites list.
+     *
+     * @param toAdd person to be added to the favorites list.
+     * @throws DuplicatePersonException
+     */
+    public void addFavorite(ReadOnlyPerson toAdd) throws DuplicatePersonException {
+        favoritePersons.add(toAdd);
```

Comment:
`favoritePersons` is an `ArrayList`; will it throw a `DuplicatePersonException`? What if you make it a `UniquePersonList` instead?